build-serbian.sh

```diff
@@ -25,3 +25,7 @@ rm ./_site/sr/en.html
 mkdir ./_site/ru
 cp ./sr/ru.html ./_site/ru/index.html
 rm ./_site/sr/ru.html
+
+mkdir ./_site/pt-br
+cp ./sr/pt-br.html ./_site/pt-br/index.html
```

Comment:
Let's create this file, just copy-paste `en.html`, it should just contains redirection to https://lettersfromsarajevo.com/pt-br/
```
./sr/pt-br.html
```